include/tilck/kernel/paging.h

```diff
@@ -32,8 +32,17 @@
  * These MACROs can be used for the linear mapping region in the kernel space.
  */
 
-#define KERNEL_PA_TO_VA(pa) ((void *) ((ulong)(pa) + KERNEL_BASE_VA))
-#define KERNEL_VA_TO_PA(va) ((ulong)(va) - KERNEL_BASE_VA)
+#if defined(TESTING) || defined(KERNEL_TEST)
```

Comment:
This is wrong. There is no need for this #ifdef and only `BASE_VA` needs to be used. The name of the macro is wrong now. It was good in the past when we used `KERNEL_BASE_VA`. Now, it should be something like `PA_TO_LIN_VADDR`. The other name too: `KERNEL_VA_TO_PA` should be `LIN_VADDR_TO_PA` or something like that.